README.md

```diff
@@ -1,2 +1,120 @@
+Each amount should translate to ML (including ML itself = 1).
+Use the same constant syntax.
+For example:
+```
+public static final float TSP = 5;
+public static final float ML = 1;
+```
+
+## Get ready for conversions (VERY EASY)
+
+Create a class CookingMeasurementConverter, which has a static method `CookingMeasurement convert(CookingMeasurement originalMeasurement, String unit)`
+
+## Basic conversion (EASY)
+
+For now, `convert` just supports TSP to ML conversion.
+
+## More conversions (EASY)
+
+Add the remaining conversions to ML: OZ, POUND, GALLON, QUART, PINT, CUP, FL_OUNCE, TBSP, TSP.
+
+Hint: create a function which uses a switch statement for checking for unit and then returning the right amount for it.
+That function should be place in `Amounts` class.
+
+## Console conversion (MEDIUM)
+
+Create a class called `ReceipePrompt`, with a public method `void run()`, which calls two private methods: 
+`void promptAmount()` and `void promptUnit()` that reads both the amount and the unit for cooking measurement.
+A user should see a prompt to enter those and in case invalid (not existing unit, negative amount) is entered, 
+a retry to enter it should be asked again.
+
+Hint: give user freedom and make sure capitalization does not matter. Use `toUpper()`.
+
+Hint: to check if the unit is valid, use the `getValues` function from the `Units` class.
+
+Hint: you can add one more function in the `Units` class `boolean isValid(String unit)` to check if the inputed String is a valid unit.
+
+## Two-way conversion (HARD)
+
+In `convert` function, add support for converting from/to every known unit described previously. 
+
+Hint: use the function from *More Conversions* step (with a switch statement) to getAmount
+amount of original measurement and measurement you need to convert it to, take their ration and multiple it original amount from that ratio. 
+
+## Menu (MEDIUM)
+
+Update `ReceipePrompt` class `Run()` method. It no longer does the two prompts.
+Instead, it prints a menu first:
+
+```
+1 - basic conversion of a measurement
+2 - receipe processing
+```
+
+User is prompted to select either option 1 or 2. Selecting option 1 does the same as before (basic conversion).
+Selecting option 2 prints "to be implemented" in the console (in red).
+
+Hint: don't forget to reset the console color after it prints text in red.
+
+## Recipe conversion (VERY HARD)
+
+Implement the option 2 in menu selection.
+User pastes a recipe in the console and it should convert that to have all measurements in ML.
+
+Algorithm:
+1. Split input recipe by " " (space).
+2. Run a for loop for every word split
+3. If a word is a number...
+4. ..Check what is the next word...
+5. ..If the word that follows a number is a known measurement (using the function that you already made)...
+6. ..Go back to the number, parse it, and convert it to ML...
+7. ..And change the word after number to ML.
+
+## Future- refactor using enum (MEDIUM)
+
+TBD
```

Comment:
Should have merged to homework branch.
![image](https://user-images.githubusercontent.com/40486932/98452285-747a2c80-2156-11eb-9e53-64c3bcad7095.png)
not main